.github/workflows/release.yml

```diff
@@ -24,9 +26,12 @@ jobs:
         roleId: ${{ secrets.VAULT_ROLE_ID }}
         secretId: ${{ secrets.VAULT_SECRET_ID }}
 
-    - name: Set version if tags
-      run: echo "VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-      if: startsWith(github.ref, 'refs/tags/v')
+    - name: Set version if tag exists
```

Comment:
I think something is wrong... https://github.com/elastic/opbeans-node/pull/227 was merged and replace `name: Set version if tags` with a specific GitHub action for creating docker metadata

Reply:
True, and the workflow is quite different. I'm going to close this then.